docs/src/Groups/pcgroup.md

```diff
@@ -132,6 +130,7 @@ map_word(g::Union{PcGroupElem, SubPcGroupElem}, genimgs::Vector; genimgs_inv::Ve
 ## Functions for (subgroups of) pc groups
 
 ```@docs
+pc_group(G::T) where T <: Union{Group, FinGenAbGroup}
```

Comment:
Is this the appropriate place, though? This is a function which is not primarily meant to be called on subgroups of pc groups, as the section would suggest. Rather it is meant to be called on arbitrary groups. (Same for `fp_groups`.)

Perhaps at the end of the "Basic Creation" section would fit better? Together with a blurb text, like: "Another way to construct a pc group is by converting a group of another type into a pc group, either via `isomorphism` (TODO: make this a `@ref`), or via `pc_groups`: